src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
@sagikazarmark I do agree that this inconstancy is anyoing, even the scalar typehints proposal uses "integer" in some of it's written descriptions...

But everyone seems to be using "int" and "bool". Symfony added that to their cs guidelines a few months back too.